code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -361,7 +361,7 @@
 	if((body_zone != "head" && body_zone != "chest"))
 		should_draw_gender = FALSE
 
-	if(status == BODYPART_ORGANIC)
+	if(status == BODYPART_ORGANIC || icon == 'icons/mob/augmentation/ipc.dmi' ) // S.N.O.W.F.L.A.K.E.
```

Comment:
Oh. God.

Two ways to go about this:

a) just roll with regular robotic bodypart sprites. We'd want those to be the same thing as regular cybernetic limbs everyone can add to themselves anyway, so it will help us avoid confusion and the IPC sprites are shit anyway..

b) Add a specific variable on the species datum that links to the icon file that should be checked when you're forging the icons.

Reply:
Okay, this is going to take some explanation.

I had to make IPCs spawn with a full set of augmented limbs in order to have them properly inherit the `BODYPART_ROBOTIC` flag, and everything attached to that. Repair, EMP effects, sounds, limb properties, etc.

Augmented limbs (any their few flavors of Grey/Standard/Miner/Sec/Eng) ignore all inherited color. This is to prevent an augmented limb from looking flesh colored. It makes sense in the case of the rest of the augmented limbs, obviously, but not for IPCs. This solution I've used here is _fucking terrible_ and I am ashamed of it, but I was unable to figure out a better solution to getting augmented limbs to render with color.

In regards to the two ways:

a) MUH SCREENS AND ANTENNAS - the standard `BODYPART_ROBOTIC` limbs are for augmentation, not full robits. Some of those can be seen [here](https://github.com/tgstation/tgstation/pull/28977). They're still very human shaped. 

b) This is already done in their `on_species_gain` shenanigans. `Z.icon = 'icons/mob/augmentation/ipc.dmi'`

If I can figure out a way to make their icons render with color while retaining `BODYPART_ROBOTIC`, this goes away.